examples/abstract-objects/abstract-objects.description

```diff
@@ -0,0 +1,3 @@
+// In Ballerina, objects can be abstract. These objects only describes the type of
+// each field and method. These objects cannot be initialized, and does not have a
+// default value.
```

Comment:
Minor changes:
"// In Ballerina, objects can be abstract. Abstract objects should only describe
// each field type and method. An abstract object cannot be initialized, and does not have a
// default value."